src/libs/actions/Report.js

```diff
@@ -693,7 +693,6 @@ function deleteReportComment(reportID, reportAction) {
     // If we are deleting the last visible message, let's find the previous visible one (or set an empty one if there are none) and update the lastMessageText in the LHN.
     // Similarly, if we are deleting the last read comment we will want to update the lastVisibleActionCreated to use the previous visible message.
     let optimisticReport = {
-        lastMessageText: '',
```

Comment:
revert, no need to change the the last message.

Reply:
Also this https://github.com/Expensify/App/issues/17164#issuecomment-1506972128.

> 3. (My proposal) Add a logic inside [deleteReportComment()](https://github.com/Expensify/App/blob/bcada91a733f8e821deb08bf23249668f09ec80e/src/libs/actions/Report.js#L674) function to only update the last message only if it is a valid text (not an empty string). This will prevent the LHN preview text to update when there is no message history.

Reply:
This is not required. It can be reported as a regression (message not being deleted from LHN if it's the last message).

Reply:
@s77rt  Updated. Please have a look!